dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -20,21 +20,22 @@ jwtSso.desc=Information about configuring JWT Single Sign On.
 cookieName=Name of cookie
 cookieName.desc=Name of the cookie that is used to store the JWT token.
 
-setCookiePathToWebAppContextPath=Set cookie path to context path
-setCookiePathToWebAppContextPath.desc=Set the cookie path of the SSO cookie to the context path of the application that is being authenticated, which limits the use of this cookie to only this app.
+setCookiePathToWebAppContextRoot=Set cookie path to context path
+setCookiePathToWebAppContextRoot.desc=Set the cookie path of the JWT cookie to the context path of the application that is being authenticated, which limits the use of this cookie to only this app.
 
 includeLtpaCookie=Include LTPA cookie
-includeLtpaCookie.desc=After successful authentication with a JWT token, include an LTPA cookie in addition to the SSO cookie.
+includeLtpaCookie.desc=After successful authentication with a JWT token, include an LTPA cookie in addition to the JWT cookie.
 
-fallbackToLtpa=Fall back to LTPA cookie
-fallbackToLtpa.desc=If the SSO cookie is missing or invalid, attempt to process an LTPA cookie.
+#initially this property was exposed as fallbackToLtpa but now is useLtpaIfJwtAbsent
+fallbackToLtpa=Use LTPA cookie if JWT cookie is not present
+fallbackToLtpa.desc=If the JWT cookie is missing, attempt to process an LTPA cookie if it is present. 
 
 jwtBuilderRef=JWT builder reference
 jwtBuilderRef.desc=A reference to the JWT Builder configuration element in server.xml that describes how to build the JWT token.
 
-setCookieSecureFlag=Set secure flag on SSO cookie
-setCookieSecureFlag.desc=Set the secure flag on the SSO cookie to send it only over a secured connection. 
+setCookieSecureFlag=Set secure flag on JWT cookie
+setCookieSecureFlag.desc=Set the secure flag on the JWT cookie to send it only over a secured connection. 
 
-jwtConsumerRef=JWT consumer reference
-jwtConsumerRef.desc=A reference to the JWT Consumer configuration element in server.xml that describes how to consume the JWT token.
+mpjwtConsumerRef=MicroProfile JWT reference
+mpjwtConsumerRef.desc=A reference to the MicroProfile JWT token configuration element in the server.xml that describes how to process the JWT token.
```

Comment:
Please add the word "file" after "server.xml." These extra descriptive words can be helpful in translation. :) Thank you!

Reply:
We ended up removing this attribute from the configuration. I updated properties file to reflect this change.
Thanks.